Update push changes to global styles label text to "apply globally"

andrewserong opened the pull request:
<!-- Thanks for contributing to Gutenberg! Please follow the Gutenberg Contributing Guidelines:
https://github.com/WordPress/gutenberg/blob/trunk/CONTRIBUTING.md -->

## What?
<!-- In a few words, what is the PR actually doing? -->

Fixes #46928

Renames "Push changes to global styles" to "Apply globally" at the block level in the site editor.

## Why?
<!-- Why is this PR necessary? What problem is it solving? Reference any existing previous issue(s) or PR(s), but please add a short summary here, too -->

As raised in #46928, this avoids referring to Global Styles as a proper noun, and instead the label text describes the action.

## How?
<!-- How is your PR addressing the issue at hand? What are the implementation details? -->

* Update the button label and help text.
* Update the snackbar notification text to say "styles updated".

## Testing Instructions
<!-- Please include step by step instructions on how to test this PR. -->
<!-- 1. Open a Post or Page. -->
<!-- 2. Insert a Heading Block. -->
<!-- 3. etc. -->

1. Open up the site editor, and add a block that can be styled by global styles (e.g. Button block)
2. At the block level, select the button and expand the Advanced area
3. See that the label of the button under Styles now says "Apply globally"
4. Once styles are applied the snackbar notification text should also read "styles updated" instead of "pushed"

## Screenshots or screencast <!-- if applicable -->

| Before | After |
| --- | --- |
| <img width="300" src="https://user-images.githubusercontent.com/26996883/210913526-77cc2bf3-1482-477b-a7da-8e7af09b2867.png" /> | <img width="300" src="https://user-images.githubusercontent.com/14988353/211428568-5fb45e8b-1618-49cf-a3cb-eae3186b78f7.png" /> |

The snackbar notification text is also updated:

<img width="350" src="https://user-images.githubusercontent.com/14988353/211428540-a10de0a0-70e8-4a1e-9d71-3adc8952574a.png" />

andrewserong committed:
Global styles: Update label of button that pushes local block styles to global styles

andrewserong committed:
Update label to say Apply globally

andrewserong committed:
Updates tests and snackbar notification text

noisysocks commented:
I've no issues with it. cc. @jasmussen for his thoughts as he worked on the design of this feature.

Another suggestion was to make the button read "Apply to all Heading blocks". (_Heading_ would be replaced with the selected block.) I think it might have caused issues with overflowing text though. What do you think?

andrewserong commented:
Thanks for taking a look!

> I think it might have caused issues with overflowing text though

Oh, that's a good point, I see from your discussion with Ramon that the button looks a bit busy if it winds up being forced to wrap: https://github.com/WordPress/gutenberg/pull/46446#discussion_r1050302487

Happy to go with whatever folks, like, though... let's see what the designers think!

jameskoster commented:
I think it's best to keep the button label short, especially since the help text beneath describes the full behaviour. 

For the Snackbar, perhaps we can go even shorter: "Button styles updated". What do you think?

juanfra commented on packages/edit-site/src/hooks/push-changes-to-global-styles/index.js:
```diff
@@ -137,7 +137,7 @@ function PushChangesToGlobalStylesControl( {
 				onClick={ pushChanges }
 			>
-				{ __( 'Push changes to Global Styles' ) }
+				{ __( 'Apply globally' ) }
```
Looking great!

We may also want to update the description (line 127) as [discussed here](https://github.com/WordPress/gutenberg/issues/46928#issuecomment-1375557951)

andrewserong committed:
Update help text and snackbar text

andrewserong commented:
Thanks for the feedback, folks! I've updated the help text to use "Apply" instead of "Move", and updated the snackbar notification to say "{block} styles updated."

Here's how it's looking now:

| Button and help text | Snackbar notification |
| --- | --- |
| <img width="300" src="https://user-images.githubusercontent.com/14988353/211428568-5fb45e8b-1618-49cf-a3cb-eae3186b78f7.png" /> | <img width="350" src="https://user-images.githubusercontent.com/14988353/211428540-a10de0a0-70e8-4a1e-9d71-3adc8952574a.png" /> |

Let me know if you want any other changes! 🙂

jameskoster committed:
Snackbar label

jameskoster commented:
I made a small change to my own suggestion :) The Snackbar now reads:

"Button styles applied."

This harmonizes with the language in the button / description, and hopefully eliminates any confusion as to whether those changes have been _saved_ or not.

andrewserong committed:
Update test text for snackbar to match

andrewserong commented:
Thanks for updating this one @jameskoster! I've just updated the text in the e2e test to match, and will merge this in once the tests pass 🙂

jasmussen commented:
Nice one!
